src/frontend/src/pages/AdminToolsPage/AdminToolsPage.tsx

```diff
@@ -4,13 +4,47 @@
  */
 
 import PageTitle from '../../layouts/PageTitle/PageTitle';
-import AdminToolsUserMangaement from './AdminToolsUserManagement';
+import AdminToolsUserManagement from './AdminToolsUserManagement';
+import PageBlock from '../../layouts/PageBlock';
+import { NERButton } from '../../components/NERButton';
+import { Box, TextField, Tooltip } from '@mui/material';
+import { useState } from 'react';
 
 const AdminToolsPage: React.FC = () => {
+  const [days, setDays] = useState<string | null>(null);
+
+  const [disableButton, setDisableButton] = useState(false);
+
+  const disable = () => {
+    setDisableButton(true);
+  };
+
   return (
     <>
       <PageTitle title={'Admin Tools'} previousPages={[]} />
-      <AdminToolsUserMangaement />
+      <AdminToolsUserManagement />
+      <PageBlock title={'Slack Upcoming Deadlines'}>
+        <Box gap={2} sx={{ display: 'flex' }}>
+          <Tooltip title="Enter in a positive number of days until a deadline, where all projects with this deadline will be slacked">
```

Comment:
- work packages, not projects
- work packages that are due on that day or before will be slacked
- so if the day is 5, all work packages that are due 5 days from now or before (even into the past from today) will generate a slack message